README.md

```diff
@@ -22,9 +22,36 @@ brew install skycoin-cx
 # Testing changes
 The following process can be used to test changes made to formula:
 1. Ensure the target formula has already been installed on your system. I will use `skycoin-cx` to demonstrate.
-2. Fork this repo on Github and clone it locally to your machine.
+2. Fork the repo on Github and clone it locally to your machine.
 3. Create a new branch for your changes.
 4. Make your changes to the target formula and commit them.
+5. At this point you can choose to either test from your own repo - which is probably the easist, or test using localised insall.
+    - [Repo based testing](#repo-based-testing)
+    - [Localise testing](#localised-testing)
+
+## Repo based testing
+5. Push the changes back to your repo - this is where we will test from.
+6. Uninstall `skycoin-cx` (and any other formula from the Skycoin tap):
+```bash
+brew uninstall skycoin-cx
+```
+7. Untap `Skycoin`:
+```bash
+brew untap skycoin/skycoin
```

Comment:
`skycoin/homebrew-skycoin`

Reply:
I will check - but I’m sure when you untap it’s actually skycoin/skycoin

Reply:
I have checked this and Homebrew drops the `homebrew` prefix on the repo when you tap it. So both of these are considered valid:
- `skycoin/homebrew-skycoin`
- `skycoin/skycoin`

When you run `brew tap` it shows as:
- `skycoin/skycoin`

I will update to ensure both statements are consistent and use `skycoin/skycoin` as this is what is reported by `Brew` once tapped.